src/cmd/nsfs.js

```diff
@@ -14,8 +14,6 @@ const fs = require('fs');
 const util = require('util');
 const minimist = require('minimist');
 
-require('../server/system_services/system_store').get_instance({ standalone: true });
```

Comment:
@romayalon if we will remove it from here then later the `endpoint` will initialize the system_store and that too without `standalone: true` being set (if `LOCAL_MD_SERVER` is set to `true`). 

I think that's why this was added here before other things are included which can potentially bring system_store.

Reply:
@tangledbytes I thought that too, but in NC NSFS - LOCAL_MD_SERVER is false, so I guess it's related to tape only? 
I tested it (LOCAL_MD_SERVER is not set) and it worked
do you know if there are plans of using data buckets on tape soon?

Reply:
A recent POC that Martin did, did use data buckets. Lemme quickly check with them again...

Reply:
so sounds like we need it, or maybe in the future, will it work if I'll require systemStore when LOCAL_MD_SERVER=true?

Reply:
Yeah! That sounds good!